helm_deploy/hmpps-template-kotlin/values.yaml

```diff
@@ -18,7 +18,6 @@ generic-service:
   env:
     JAVA_OPTS: "-Xmx512m"
     SERVER_PORT: "8080"
-    SPRING_PROFILES_ACTIVE: "logstash"
```

Comment:
unnecessary - was used originally when we had multiple logging configurations for running locally and also running in kubernetes